core/src/main/java/org/elasticsearch/rest/action/admin/indices/RestClearIndicesCacheAction.java

```diff
@@ -85,6 +86,9 @@ public static ClearIndicesCacheRequest fromRequest(final RestRequest request, Cl
             if (parseFieldMatcher.match(entry.getKey(), Fields.QUERY)) {
                 clearIndicesCacheRequest.queryCache(request.paramAsBoolean(entry.getKey(), clearIndicesCacheRequest.queryCache()));
             }
+            if (parseFieldMatcher.match(entry.getKey(), Fields.REQUEST_CACHE)) {
+                clearIndicesCacheRequest.requestCache(request.paramAsBoolean(entry.getKey(), clearIndicesCacheRequest.requestCache()));
```

Comment:
Nope, it was caught by a doc test that failed because the parameter `request_cache` went unconsumed on a call to `/_cache/clear`.